dist/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/1dea6ac3ad.js" crossorigin="anonymous"></script>
+  <title>To-Do list</title>
+<script defer src="main.js"></script></head>
+
+<body>
+  <main>
+    <p id="header">Today's To Do <button id="refresh-list"><i class="fa-solid fa-arrows-rotate"></i></button></p>
+
+    <input type="text" id="new-task" placeholder="Add to your list...">
+
+    <ul id="todo-list">
+    </ul>
+
+    <p id="clear-completed">Clear all completed</p>
```

Comment:
- [ ] Please replace `<p>` tags used used at line 14 and line 21, with collect HTML tag

Reply:
thank you, I will change it 👍🏻